src/SFA.DAS.Apprenticeships.Web/Controllers/ChangeOfPrice/ChangeOfPriceEmployerController.cs

```diff
@@ -153,9 +154,9 @@ public async Task<IActionResult> CancelPriceChange(string employerAccountId, str
     public async Task<IActionResult> ApproveOrRejectPriceChangePage(string employerAccountId, string apprenticeshipHashedId, string ApproveChanges, string rejectReason)
     {
         var apprenticeshipKey = await _apprenticeshipService.GetApprenticeshipKey(apprenticeshipHashedId);
-        if (apprenticeshipKey == default)
+        if (apprenticeshipKey == Guid.Empty)
         {
-            _logger.LogWarning($"Apprenticeship key not found for apprenticeship with hashed id {apprenticeshipHashedId}");
+            _logger.LogWarning("Apprenticeship key not found for apprenticeship with supplied apprenticeshipHashedId");
```

Comment:
Where losing the hashId in the logs here

Reply:
That was intentional, logs whatever the user puts in the url, could be a data protection issue